### DISCUSSION TITLE
Xcode 14 beta 1 - CasePath prefix / syntax

### DISCUSSION THREAD

**Discussion by jtouzy** [pointfreeco/swift-case-paths#77] [General]
Hello guys,

It's not really a library-related bug but we should submit Feedback at Apple for an editor bug.
Since regex has been introduced in the new Xcode, all indentation / coloration stuff is totally broken when we use case paths.

For example :
<img width="400" alt="Screenshot 2022-06-07 at 16 44 29" src="https://user-images.githubusercontent.com/5709133/172501623-6311d364-0597-4f26-8116-913ac970bfc5.png">

All the coloration below the case path is broken, and you can see the cursor in the screenshot : it's indenting there just when we hit enter after the function... Totally weird.

And it's even worse in reducers with multiple pullbacks, I let you imagine...

**Comment by stephencelis** [comment]
Hi @jtouzy! We try to keep issues focused on library-related that we can fix. There is a similar discussion at #75. Hopefully Apple will resolve it before the next beta.

I'm going to convert this to a discussion for now.

**Reply by jtouzy** [reply]
Ok it's a full duplicate, didn't take time to open Discussions tab ... sorry for that. We can close this.
